hack/lib/cleanup.sh

```diff
@@ -94,8 +94,8 @@ function os::cleanup::tmpdir() {
         done
     fi
 
-		# delete any sub directory underneath BASETMPDIR
-    for directory in $( find "${BASETMPDIR}" -d 2 ); do
+    # delete any sub directory underneath BASETMPDIR
+    for directory in $( find "${BASETMPDIR}" -mindepth 1 -maxdepth 1 ); do
```

Comment:
If the intent is sub-directories, this doesn't really do it since you don't also have `-type d` but in general I would just suggest `"${BASETMPDIR}"/*` than this